docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
+
+Create a plugin
+
+**Example request**:
+
+To create a plugin named `plugin`
+
+```
+POST /plugins/plugin:latest/create HTTP/1.1
+
+Header should be set to "Content-Type: application/tar"
```

Comment:
`application/tar` is indeed not registered at http://www.iana.org/assignments/media-types/media-types.xhtml#application. Unregistered mime types are considered "experimental" and have an `x-` prefix. So it makes sense to replace with `application/x-tar`. 

Ofcourse curl recognizes  both types. Eg. `curl -H "Content-Type: application/tar" -X POST http://localhost:9999/v1.25/plugins/testplugin/create   --data-binary "@/tmp/pluginDir/context.tar"`